welcome/about-kueue.adoc

```diff
@@ -0,0 +1,53 @@
+:_mod-docs-content-type: ASSEMBLY
+include::_attributes/common-attributes.adoc[]
+[id="about-kueue"]
+= About {product-title}
+:context: about-kueue
+
+toc::[]
+
+{product-title} is a Kubernetes-native system that manages access to resources for jobs.
+{product-title} can determine when a job waits, is admitted to start by creating pods, or should be _preempted_, meaning that active pods for that job are deleted.
+
+[NOTE]
+====
+In the context of {product-title}, a job can be defined as a one-time or on-demand task that runs to completion.
+====
+
+{product-title} is based on the link:https://kueue.sigs.k8s.io/docs/[Kueue] open source project.
+
+{product-title} is compatible with environments that use heterogeneous, elastic resources. This means that the environment has many different resource types, and those resources are capable of dynamic scaling.
+
+{product-title} does not replace any existing components in a Kubernetes cluster, but instead integrates with the existing Kubernetes API server, scheduler, and cluster autoscaler components.
+
+{product-title} supports all-or-nothing semantics. This means that either an entire change is applied successfully to pods, or the entire change is rolled back, preventing partial updates and issues that could be caused by them.
+
+// Personas
+[id="about-kueue-personas"]
+== Personas
+
+Different personas exist in a {product-title} workflow.
+
+Batch administrators:: Batch administrators manage the cluster infrastructure and establish quotas and queues.
+Batch users:: Batch users run jobs on the cluster. Examples of batch users might be researchers, AI/ML engineers, or data scientists.
+Serving users:: Serving users run jobs on the cluster. For example, to expose a trained AI/ML model for inference.
+Platform developers:: Platform developers integrate {product-title} with other software. They might also contribute to the Kueue open source project.
+
+[id="about-kueue-workflow"]
+== Workflow overview
+// TODO: add diagram?
+
+The {product-title} workflow can be described at a high level as follows:
+
+. Batch administrators create and configure `ResourceFlavor`, `LocalQueue`, `ClusterQueue`, `AdmissionCheck`, and `BorrowingCohort` resources.
+. User personas create jobs on the cluster.
+. The Kubernetes API server validates and accepts job data.
+. {product-title} admits jobs based on configured options, such as order or quota. It injects affinity into the job by using resource flavors, and creates a `Workload` object that corresponds to each job.
```

Comment:
```suggestion
. {product-title} admits workloads based on configured options, such as order or quota. It injects affinity into the workload by using resource flavors.
```

IMO workloads are a bit of an implementation detail. Users do not interact with these objects. They can query those objects to see the status of the admission but I'm not sure it belongs in the about-docs.